### PR TITLE
TF-7865 Remove beta flags for  custom access for TeamProjectAccess tests

### PR DESCRIPTION
**Pull request by rberecka**
## Description

Removes skipping of Team Project Access tests using custom access now that the feature is fully GA.
Change only includes test code.

## Testing plan

CI should pass

## Output from tests
Including output from tests may require access to a TFE instance. Ignore this section if you have no environment to test against.

```
$ TFE_ADDRESS="https://example" TFE_TOKEN="example" go test ./...  -v -run TestTeamProjectAccesses  

> PASS
> ok      github.com/hashicorp/go-tfe     12.199s
...
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # UNRELEASED
-<!-- Add CHANGELOG entry to this section for any PR awaiting the next release -->
-
+* Added full support for adding and updating custom permissions to `TeamProjectAccesses`. A `TeamProjectAccessType` of `"custom"` can set various permissions applied at
+the project level to the project itself (`TeamProjectAccessProjectPermissionsOptions`) and all of the workspaces in a project (`TeamProjectAccessWorkspacePermissionsOptions`) by @rberecka [#745](https://github.com/hashicorp/go-tfe/pull/745)<!-- Add CHANGELOG entry to this section for any PR awaiting the next release -->
```
Comment (Netra2104) [inline CHANGELOG.md:3]: --> is this line supposed to above?

Comment (Netra2104) [reply CHANGELOG.md:3]: I am referring to:

**Review by Netra2104 — DISMISSED** [review]
Left 1 comment, but looks good!

File: CHANGELOG.md (hunk 1 above)
Comment (rberecka) [reply CHANGELOG.md:3]: I guess that it gets removed when there is something added to the section -- but that's maybe also a good question for the CLI team.

**Review by sebasslash — CHANGES_REQUESTED** [review]
👋 We don't need a changelog entry for flag removal. Please ensure that [references](https://github.com/hashicorp/go-tfe/blob/9073f23cd40f852bb2e61e888b992abe3996fa10/team_project_access.go#L19) to "This feature is still in BETA" are removed in the code docs (which are used to generated the godocs).

File: team_project_access.go [hunk 2]
```diff
@@ -16,7 +16,6 @@ var _ TeamProjectAccesses = (*teamProjectAccesses)(nil)
 // Enterprise API supports
 //
 // TFE API docs: Documentation will be linked once this feature is available
```
Comment (sebasslash) [inline team_project_access.go:18]: Can you add the docs URL here?

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,7 +6,7 @@
 # v1.32.0
 
 ## Enhancements
-* Added BETA support for adding and updating custom permissions to `TeamProjectAccesses`. A `TeamProjectAccessType` of `"custom"` can set various permissions applied at
+* Added support for adding and updating custom permissions to `TeamProjectAccesses`. A `TeamProjectAccessType` of `"custom"` can set various permissions applied at
```
Comment (sebasslash) [inline CHANGELOG.md:9]: I don't think we should modify previous changelog entries here.

**Review by sebasslash — CHANGES_REQUESTED** [review]
Almost there 🤞. I did give incorrect feedback in my previous review (sorry 😅 ). We should add a changelog entry: `Removed beta tags for TeamProjectAccess by @rberecka` since we _are_ making a documentation change.

**Review by nfagerlund — APPROVED** [review]

**Pull request merged**